src/pages/[slug].tsx

```diff
@@ -0,0 +1,21 @@
+import Layout from "~/components/layouts";
+import HeadsBase from "~/components/heads";
+import { api } from "~/utils/api";
+const ProfilePage = () => {
+
+  const { data } = api.profile.getUserByUsername.useQuery({username: 'maria-skygrid'})
+  console.log(data);
+  
```

Comment:
```suggestion
  if(!data) throw new Error('....')
```
early returnが使えそうです！
潜在的なバグや、必要なエラーも上書きしてしまうため`!`や`?`はなるべく減らしていきましょう！